File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> sum;
        for(int i=0; i < nums.size(); i++) {
            for(int j=0; j < nums.size(); j++) {
                if(i != j && nums[i] + nums[j] == target) {
                    if (i < j) {
                        sum.push_back(i);
                        sum.push_back(j);
                    } else {
                        sum.push_back(j);
                        sum.push_back(i);
                    }

                    return sum;
                }
            }
        }
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set(nums)
        best = 0
        for i in nums:
            if i-1 not in s:
                length = 1
                while i + length in s:
                    length += 1
                best = max(length, best)
        return best
        